src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx

```diff
@@ -26,13 +26,14 @@ export function getStaticProps(context) {
 
 There are multiple ways you can implement upload functionality for your app and upload files to your storage bucket.
 
-<BlockSwitcher>
-<Block name="Storage Manager UI Component">
+## Storage Manager UI Component
+
 Upload files from your app in minutes by using the cloud-connected Storage Manager UI Component. 
 
 ```bash
 npm install @aws-amplify/ui-react-storage aws-amplify
 ```
+
 ```tsx
 import { StorageImage } from '@aws-amplify/ui-react-storage';
```

Comment:
Shouldn't this be importing `StorageManager` instead of `StorageImage`?

Reply:
Thanks for pointing it out Arun, I have made the update. This code sample is also going to change and I will be including that in a different PR